README.md

```diff
@@ -78,6 +78,15 @@ Follow the given instructions for Login into the app.
 
 To run the tests run: `python manage.py test`.
 
+## Project Documentation
```

Comment:
Can you remove the word `project` from this?

Reply:
Sure